### PR TITLE
36_expand_device_list

### PR DESCRIPTION
**Pull request by cjkerr128**
doubled size of misc. device space, new sensors to be flown the semester, and SPOT and ARPS

### CODE REVIEW

**Review by thomasmholder — CHANGES_REQUESTED** [review]
This looks really good, but I think we could go ahead and add a bunch of MISC_TEST IDs, as we seem to go through those pretty quickly. 

We also need to bump the versioning, I think this could arguably be either a minor or major version, depending on how the validators work.

**Comment by thomasmholder** [comment]
Oh, and before I forget, APRS should have an APRS_API and an APRS_DEVICE (or something along those lines) ID, as both of those are valid but distinct sources.

**Comment by thomasmholder** [comment]
Don't worry about the versioning, just add the other stuff and you're good to go.

**Review by thomasmholder — APPROVED** [review]
Looks good to me

File: EosLib/device.py [hunk 1]
```diff
@@ -38,3 +38,32 @@ class Device(IntEnum):
...
+    MISC_CAMERA_3 = 48
+    MISC_CAMERA_4 = 49
+    MISC_5 = 50
+    MISC_6 = 51
+    MISC_7 = 52
+    MISC_8 = 53
+    MISC_TEST_4 = 54
+    MISC_TEST_5 = 55
+    MISC_TEST_6 = 56
+    APRS_API = 57
```
Comment (DarylDohner) [inline EosLib/device.py:62]: This duplicates ARPS_API = 37

**Review by DarylDohner — CHANGES_REQUESTED** [review]
see comment

**Review by DarylDohner — APPROVED** [review]

**Pull request merged**